service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/DigitalTwinAsyncClient.java

```diff
@@ -23,34 +23,41 @@
 import com.microsoft.azure.sdk.iot.service.digitaltwin.models.DigitalTwinUpdateRequestOptions;
 import com.microsoft.rest.*;
 import com.microsoft.rest.serializer.JacksonAdapter;
-import lombok.Builder;
-import lombok.NonNull;
-import lombok.Setter;
 import rx.Observable;
 import rx.schedulers.Schedulers;
 
 import java.util.List;
 import java.util.Objects;
 
 import static com.microsoft.azure.sdk.iot.service.digitaltwin.helpers.Tools.*;
-import static lombok.AccessLevel.PACKAGE;
 
 public class DigitalTwinAsyncClient {
-    @Setter(PACKAGE)
     private DigitalTwinsImpl digitalTwin;
     private static ObjectMapper objectMapper = new ObjectMapper();
 
-    /***
-     * Creates an implementation instance of {@link DigitalTwins} that is used to invoke the Digital Twin features
-     * @param connectionString The IoTHub connection string
-     */
-    @Builder(builderMethodName = "buildFromConnectionString", builderClassName = "FromConnectionStringBuilder")
-    public DigitalTwinAsyncClient(@NonNull String connectionString) {
+     DigitalTwinAsyncClient(String connectionString) {
```

Comment:
call it out as private explicitly?

Reply:
It was not done so in track2. Do you know why?